PDR-0008-Maintainer-Committee/README.md

```diff
@@ -0,0 +1,52 @@
+Category: Guidelines
+Status: Pending
+Authors:
+    - Christina Gianelloni <christina@blinklabs.io> 
+Created: 2025-02-16
+---
+
+## Context
+The [Statutes](https://github.com/pragma-org/PDRs/blob/main/.incorporation/20240422_Attachment_Statutes_Statuten_PRAGMA_signed.pdf) clearly call out the need for Maintainer Committees and Leads therein. However that information is spread throughout the statutes and at times leaves room for ambiguity. They organize, manage and implement a software project and will need guidance on the expectations and requirements within PRAGMA. 
+
+## Motivation
+Clearly defining the structure and responsibilities of the Maintainer Committees and their Leads are essential for accountability, transparency and efficiency.
+
+## Decision
+Maintainer Committees
+- Composition: The Maintainer Committee comprises individuals who have demonstrated consistent contributions to a PRAGMA project. The Lead appoints the other members. Thereafter members are appointed by invitation, from an existing member, based on merit and commitment.
+- Structure: They report to the Administrative Board in phase 1. In phase 2 Project Committees will be implemented and assume that reporting role.
+- Responsibilities:  
+  - Code Review and Quality Assurance: Ensuring that all code contributions meet the project's standards and adhere to PRAGMA policies.
+  - Issue Triage: Prioritizing and addressing issues reported by the community.
+  - Release Management: Coordinating and managing project releases.
+  - Documentation: Maintaining and updating project documentation.
+  - Community Engagement: Interacting with users and contributors to foster a healthy community.
+  - Funding: Procure their own funding and be transparent about it to all parties. 
+
+Maintainer Committee Leads
+ - Selection: Maintainer Committee Leads are appointed by the Administrative Board. 
+ - Responsibilities:
+   - Initial Members: Upon appointment, the Lead appoints the other initial members. 
+   - Initial Setup: Document the Committees organization, responsibilities and authority and submit it to the Administrative Board for approval expeditiously upon appointment. 
+   - Facilitation: Organizing and leading MC meetings and discussions.
+   - Conflict Resolution: Addressing and mediating disputes within the MC or among contributors.
+   - Delegation: Assigning tasks and ensuring fair distribution of responsibilities among MC members.
+   - Representation: Serving as the primary liaison between the MC and the Administrative Board or other committees. 
+     - They will work with the Administrative Board regarding services, such as Corporate housekeeping support, Intellectual Property support, etc. 
+     - They will ensure the MC is in compliance with PRAGMA rules and regulations, to include licensing, security, etc. 
+   - Reports: Submitting quarterly reports to the Administrative Board, due 5 days before the end of a quarter.
+
+
+## Consequences
```

Comment:
We currently have no process of removing a Lead if they are unresponsive or lose the trust of their members. I think we should add a way to remove MC Leads, if such a case happens